<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_del</class>
 <widget class="QDialog" name="form_del">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	height: 40px;
	font-family: 'Montserrat', sans-serif;
	font-size: 11px;
	text-transform: uppercase;
	font-weight: 600;
	color: rgb(205, 92, 92);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}

QPushButton:hover{
	height: 75px;
	border: 1px solid rgba(205, 92, 92, 1) ;
	border-radius: 5px;
	font-family: 'Montserrat', sans-serif;
	font-size: 11px;
	text-transform: uppercase;
	font-weight: 600;
	color: rgb(205, 92, 92);
	background-color: rgba(255, 255, 255, 0);
}

QPushButton:pressed{
	background-color: rgba(205, 92, 92, 1);
	color: white;
	border:2px solid;
	border: 1px solid rgba(205, 92, 92, 1) ;
	border-radius: 5px;
	font-family: 'Montserrat', sans-serif;

}

QLabel{
	font-family: 'Montserrat', sans-serif;
	font-size: 14px;
	color: rgb(71, 74, 81);
}

QLineEdit{
	border: 1px solid rgb(0, 188, 188);
	border-radius: 3px;
}

QComboBox{
	border: 1px solid rgb(0, 188, 188);
	border-radius: 3px;
}

 QComboBox::down-arrow {
	background-color: white;
	border: none;
 }

QComboBox::drop-down {
    subcontrol-origin: margin;
	border: none;
 }
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Delete records</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Table name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Table ID:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QPushButton" name="but_del">
       <property name="text">
        <string>DELETE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="but_can_2">
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
